docs/spec/mint/state.md

```diff
@@ -8,8 +8,9 @@ The minter is a space for holding current inflation information.
 
 ```golang
 type Minter struct {
-	InflationLastTime time.Time // block time which the last inflation was processed
-	Inflation         sdk.Dec   // current annual inflation rate
+	LastUpdate       time.Time // time which the last update was made to the minter
+	Inflation        sdk.Dec   // current annual inflation rate
+	AnnualProvisions sdk.Int   // current annual exptected provisions
```

Comment:
```suggestion
	AnnualProvisions sdk.Int   // current annual expected provisions
```

Reply:
darn made this change before seeing this suggested change... lol thanks! (luv it)